packages/fxa-settings/src/lib/context/context-transforms.ts

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Functions for transforming class property names to context keys.
+ */
+export const ContextKeyTransforms = {
```

Comment:
We might need other transforms on the values as well. For example coercing encoded URI components to a typed object. I think this will be more obvious once tests are ported.